pkg/yang/entry.go

```diff
@@ -110,6 +110,13 @@ type RPCEntry struct {
 	Output *Entry
 }
 
+// A ListAttr is associated with an Entry that represents a List node
+type ListAttr struct {
+	MinElements *Value // leaf-list or list MUST have at least min-elements
+	MaxElements *Value // leaf-list or list has at most max-elements
+	OrderedBy   *Value // order of entries determined by "system" or "user"
+}
+
```

Comment:
Ahh, yes, I missed the "deleted lines" in the left column (am too used to seeing moved lines marked as moved).